src/scheduled_tasks.rs

```diff
@@ -48,32 +48,68 @@ pub fn setup(
 
   // Update active counts every hour
   let url = db_url.clone();
-  scheduler.every(CTimeUnits::hour(1)).run(move || {
-    let mut conn = PgConnection::establish(&url).expect("could not establish connection");
-    active_counts(&mut conn);
-    update_banned_when_expired(&mut conn);
-  });
+  scheduler
+    .every(CTimeUnits::hour(1))
+    .run(move || match PgConnection::establish(&url) {
+      Ok(mut conn) => {
+        active_counts(&mut conn);
+        update_banned_when_expired(&mut conn);
+      }
+      Err(e) => {
+        error!(
+          "Failed to establish db connection for active counts update: {}",
+          e
+        );
+      }
+    });
```

Comment:
This way is extremely verbose. Using `inspect_err` would be ideal but its not stable yet. So I would use `map_err(/*log here*/).ok()` instead.

Reply:
Done 👍